main.go

```diff
@@ -21,6 +22,13 @@ import (
 	"github.com/honeycombio/libhoney-go/transmission"
 )
 
+const (
+	// Waiting too long to send a batch of events can be
+	// expensive in Lambda. It's reasonable to expect a
+	// batch send to complete in this amount of time.
+	defaultBatchSendTimeout = time.Second * 15
```

Comment:
Devil's advocate: is there ever a scenario where 15 seconds is for some reason _not_ considered reasonable? Basically, are we breaking for anyone who is commonly waiting and expecting a wait of >15 seconds for... something? I assume not, but wanted to ask the question just in case.

Reply:
I honestly can't think of any reason why anyone who actually has enough volume to worry about the cost of their lambda would be upset about this.